src/schema/v2/artwork/index.ts

```diff
@@ -57,9 +57,10 @@ import { ArtworkContextGrids } from "./artworkContextGrids"
 import { PageInfoType } from "graphql-relay"
 import { getMicrofunnelDataByArtworkInternalID } from "../artist/targetSupply/utils/getMicrofunnelData"
 import { InquiryQuestionType } from "../inquiry_question"
+import { priceDisplayText } from "lib/moneyHelpers"
 
 const has_price_range = (price) => {
-  return new RegExp(/\-/).test(price)
+  return new RegExp(/-/).test(price)
```

Comment:
what is this change doing?

Reply:
That's unnesesary escaping, it was displayed as a warning in IDE, so i removed it.